src/helpers/delegationV2/delegation.d.ts

```diff
@@ -0,0 +1,20 @@
+import { DelegateWithPercent } from '@/helpers/interfaces';
```

Comment:
Our naming convention would be to name this file `types.ts`

Reply:
fixed in [18e49c7](https://github.com/snapshot-labs/snapshot/pull/4492/commits/18e49c74c87feea8d7e4030441dcea6ff532ad34)